doc/changelog.rst

```diff
@@ -1,6 +1,26 @@
 Changelog
 =========
 
+.. rubric:: 1.5
+
+This version mainly focuses on performance. Improvements of 5× have been
```

Comment:
Since I'm now sensitised by the recent katpoint discussions: did you mean to use a Unicode '×' in 5x? :-D

Reply:
Yes. It's 5 times faster, not 5 ex faster.

Reply:
It does look pretty 🙂